README.md

```diff
@@ -625,6 +626,15 @@ loading only those files that are actually bundled by webpack, as well as any `.
 by the `tsconfig.json` settings. `.d.ts` files are still included because they may be needed for
 compilation without being explicitly imported, and therefore not picked up by webpack.
 
+#### useCaseSensitiveFileNames
+| Type | Default Value |
+|------|--------------|
+| `boolean` | determined by typescript based on platform |
+
+The default behavior of `ts-loader` is to act as a drop-in replacement for the `tsc` command,
+so it respects the `useCaseSensitiveFileNames` set internally by typescript. The `useCaseSensitiveFileNames` option modifies this behavior,
```

Comment:
Is it worth mentioning here what the default `useCaseSensitiveFileNames` behaviour of TypeScript is here?

Reply:
I stuck `determined by typescript based on platform` in the default above. Windows is case-insensitive and all other platforms are tested at runtime. Happy to include that if you like but I didn't want to tie docs to the internals of typescript
https://github.com/microsoft/TypeScript/blob/master/src/compiler/sys.ts#L1427

Reply:
Oh no - that works well enough I think.